application/src/main/java/org/wecancodeit/serverside/model/Comment.java

```diff
@@ -1,4 +1,33 @@
 package org.wecancodeit.serverside.model;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import java.util.Collection;
+@Entity
 public class Comment {
+
+    @Id
+    @GeneratedValue
+    private String commentName;
+
+    @ManyToMany
+    private Collection<Comment> comments;
+
+    @ManyToMany
```

Comment:
We probably want this as a ManyToOne: a post can have many comments, but each comment should be attached to a single post